### PR TITLE
review commit check for new branch

### PR DESCRIPTION
**Pull request by Vidhya-Arjun**
review commit and merge to master

### CODE REVIEW

File: src/test/resources/Feature/loginpage.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+some code to test the login page function..
```
Comment (Vidhya-Arjun) [inline src/test/resources/Feature/loginpage.feature:1]: Add test case to validate for empty passwords.
